CHANGELOG.rst

```diff
@@ -18,21 +18,27 @@ number of the code change for that issue.  These PRs can be viewed at:
     https://github.com/spacetelescope/drizzlepac/pulls
 
 
+3.7.1 (unreleased)
+======================
 
-3.7.0rc2 (22-Mar-2024) Infrastructure Build
-===========================================
-
-- Force the identified bad rows to be removed from the total (aka white light)
-  source catalog before the corresponding bad segments are removed from the
-  segmentation image. [#1771]
+- Removed "tophat" as a kernel option, added warnings for "gaussian" and "lanczos3" 
+  that they may not be conserving flux. [#1786]
 
 - Bug fix for mdriztab=True option in Astrodrizzle previously overwriting user inputs. [#1774]
 
 - Reverted PR #1222 allowing pixels to be filled with available data where WHT=0. [#1767]
 
+- Force the identified bad rows to be removed from the total (aka white light)
```

Comment:
FYI - We can fix up the CHANGELOG on the main branch after Drizzlepac 3.7.0 is released.  The PR#1771 was part of 3.7.0 as the change was specifically requested.